example/src/streams.ts

```diff
@@ -0,0 +1,117 @@
+
+// Load e2e environment variables into process.env
+require('dotenv').config({ path: 'example.env' });
+const fetch = require('node-fetch');
+import { getToken, checkJsonError } from '../../tapis-typescript/e2e/utils';
+import { Authenticator, Streams } from '@tapis/tapis-typescript';
+
+const version = 5;

```

Comment:
What does this version number refer to?

Reply:
I went through multiple versions of the test objects while testing the spec, I was using the version number as an easy way to update this.

Reply:
migrated to env